listenbrainz/db/mbid_manual_mapping.py

```diff
@@ -0,0 +1,87 @@
+from typing import List, Optional
+import uuid
+
+from sqlalchemy import text
+
+from listenbrainz.db.model.mbid_manual_mapping import MbidManualMapping
+from listenbrainz.db import timescale as ts
+
+def create_mbid_manual_mapping(mapping: MbidManualMapping):
+    """Save a user mapping to the database. """
+    query = """
+        INSERT INTO mbid_manual_mapping(recording_msid, recording_mbid, user_id)
+             VALUES (:recording_msid, :recording_mbid, :user_id)
```

Comment:
I think ON CONFLICT DO UPDATE would be better to allow users to update their mappings in case of error or just if a new recording was added to MB etc. You already have the required UNIQUE index in place.

Reply:
good idea, it also answers my question about how we should catch/return postgres exceptions in the case of one being added twice